stl/team_aster/s1mplebug/learning.md

```diff
@@ -1 +1,182 @@
-it's learning document.
+# task1 list源码
+
+## 前言
+
+今天大概看了下list源码（gcc4.9版本的），其实对类型萃取和分配器这里还是很模糊，对list的结构大概有了个了解，实际上list的实现为一个双向链表。
```

Comment:
You may understand in two days, there will be a special section later

Reply:
ok hope i can make it to that day 😂